### PR TITLE
FYLE-85zrk1n2yCode Coverage : Ellipses pipe

### PR DESCRIPTION
**Pull request by CODEX108**

### CODE REVIEW

File: src/app/shared/pipes/ellipses.pipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    expect(ellipsisPipe.transform(input, 5)).toEqual(expectedOutput);
+  });
+
+  it('ellipsisPipe transform() : should return the original string if val is shorter than or equal to args', () => {
+    const input = 'Hello';
+    const expectedOutput = 'Hello';
+
+    expect(ellipsisPipe.transform(input, 5)).toEqual(expectedOutput);
+
+    const input2 = 'Hello';
```
Comment (jayfyle) [inline src/app/shared/pipes/ellipses.pipe.spec.ts:40]: Try and use a larger string since the case for equal length has been covered

**Review by jayfyle — APPROVED** [review]
Update the spec, rest looks good

File: src/app/shared/pipes/ellipses.pipe.spec.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/shared/pipes/ellipses.pipe.spec.ts:40]: yes, for ellipsisPipe let's use long strings

File: src/app/shared/pipes/ellipses.pipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+describe('EllipsisPipe', () => {
+  const ellipsisPipe = new EllipsisPipe();
+
+  it('should create an instance', () => {
+    expect(ellipsisPipe).toBeTruthy();
+  });
+
+  it('ellipsisPipe transform() : should return original input value if args is undefined', () => {
+    const input = 'We are testing';
```
Comment (Dimple16) [inline src/app/shared/pipes/ellipses.pipe.spec.ts:11]: use string inputs related to Fyle app, - `Jan - 1 Report for reimbursement`

File: src/app/shared/pipes/ellipses.pipe.spec.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    expect(ellipsisPipe.transform(input, 10)).toEqual(expectedOutput);
+
+    const input2 = null;
+    const expectedOutput2 = null;
+
+    expect(ellipsisPipe.transform(input2, 10)).toEqual(expectedOutput2);
+  });
+
+  it('ellipsisPipe transform() : should return a truncated string if val is longer than args', () => {
+    const input = 'Hello World';
```
Comment (Dimple16) [inline src/app/shared/pipes/ellipses.pipe.spec.ts:29]: use long category/project names as examples

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

**Review by Dimple16 — CHANGES_REQUESTED** [review]
tests lgtm, add param and return types for transform() method

**Review by jayfyle — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**